package.json

```diff
@@ -19,7 +19,8 @@
     "lint": "eslint src --ext .ts,.js",
     "lint:fix": "eslint --fix . --ext .ts,.js",
     "verify": "npm run lint && npm run test",
-    "sandbox": "DEBUG=ERROR-*,WARN-*,INFO-*,DEBUG-*,-DEBUG-helper*,-DEBUG-util* nodemon -r dotenv/config -r tsconfig-paths/register"
+    "sandbox": "DEBUG=ERROR-*,WARN-*,INFO-*,DEBUG-*,-DEBUG-helper*,-DEBUG-util* nodemon -r dotenv/config -r tsconfig-paths/register",
+    "contracts": "rimraf \"src/contracts/gen\" && npx typechain --target=web3-v1 --outDir src/contracts/gen \"src/contracts/abi/*.json\" && prettier --write src/contracts/gen/*.ts"
```

Comment:
`\"src/contracts/abi/*.json\"` Was it not picking up without quotes?

Reply:
for some reason, it was only getting 1 ABI without that, I don't know why...